src/static/menu-win.json

```diff
@@ -81,6 +81,15 @@
                     "separator": true
                 },
                 {
+                    "id": "EXPORT_UTILITY",
+                    "shortcut": {
+                        "keyChar": true,
+                        "modifiers": {
+                            "command": true
```

Comment:
Good news: I believe the menu infrastructure should throw an error if you try to load a `command` shortcut on Windows.